tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1949,34 +1949,43 @@ def __init__(self, num_units, initializer=None, num_proj=None,
         projection matrices.
       num_proj: (optional) int, The output dimensionality for the projection
         matrices.  If None, no projection is performed.
-      number_of_groups: (optional) int, number of groups to use. If number_of_groups=1,
-        then it should be equivalent to LSTMP cell
+      number_of_groups: (optional) int, number of groups to use.
+        If number_of_groups=1, then it should be equivalent to LSTMP cell
```

Comment:
If you click the 'pencil' button in the review, you can make these changes yourself now (it's a lot faster than the back and forth of code review).  Want to give it a shot?